### PR TITLE
Create an-inspired-utterance-on-annihilation_iromi-ariyaratne.md

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "An Inspired Utterance on Annihilation"
+authors:
+  - "Iromi Ariyaratne"
```
Comment (buddhist-uni) [inline _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md:4]:
```yaml
  - analayo
editor: "Iromi Ariyaratne"
```

File: _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - "https://drive.google.com/file/d/1-jXKK2Pu0kYORqjECuJf15VjVQj6Xt9n/view?usp=sharing"
+course: thought
+tags:
+  - sutta
+  - agama
+  - anatta
+year: 2021
+journal: "Sri Lanka International Journal of Buddhist Studies" 
+publisher: 
+address: ""
```
Comment (buddhist-uni) [inline _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md:16]: ```address: "Kundasale, Sri Lanka"```

File: _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+drive_links:
+  - "https://drive.google.com/file/d/1-jXKK2Pu0kYORqjECuJf15VjVQj6Xt9n/view?usp=sharing"
+course: thought
+tags:
+  - sutta
+  - agama
+  - anatta
+year: 2021
+journal: "Sri Lanka International Journal of Buddhist Studies" 
+publisher:
```
Comment (buddhist-uni) [inline _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md:15]: SIBA

File: _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+title: "An Inspired Utterance on Annihilation"
+authors:
+  - "Iromi Ariyaratne"
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/inspiredutterance.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1-jXKK2Pu0kYORqjECuJf15VjVQj6Xt9n/view?usp=sharing"
+course: thought
+tags:
+  - sutta
```
Comment (buddhist-uni) [inline _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md:10]: `udana`

File: _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  - agama
```
Comment (buddhist-uni) [inline _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md:11]: `sa`

File: _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  - anatta
```
Comment (buddhist-uni) [inline _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md:12]: `emptiness`

File: _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+---
+title: "An Inspired Utterance on Annihilation"
+authors:
+  - "Iromi Ariyaratne"
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/inspiredutterance.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1-jXKK2Pu0kYORqjECuJf15VjVQj6Xt9n/view?usp=sharing"
+course: thought
```
Comment (buddhist-uni) [inline _content/articles/an-inspired-utterance-on-annihilation_iromi-ariyaratne.md:8]: 🤔 I'm thinking to move this to `nibbana` What do you think?

**Comment by JJM317** [comment]
I'd keep it in 'thought'. I was thinking of 'view', which could work, but
'thought' is probably better. I'm not feeling 'nibbana'

~Joe

**Review by buddhist-uni — APPROVED** [review]
Okay. Thought it is!

**Pull request merged**